test/fixtures/pipelines/pipelines.js

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import { AUTHORS, PROGRAMMING_LANGUAGES } from '../../../src/shared/constants';
+
+export const PRIMARY_PIPELINE = {
+  name: 'first pipeline',
+  description: 'pipeline description 1',
+};
+export const SECONDARY_PIPELINE = {
+  name: 'second pipeline',
+  description: 'pipeline description 2',
+};
+
+export const FIRST_ALGORITHM_INDEX = 0;
+export const SECOND_ALGORITHM_INDEX = 1;
+export const THIRD_ALGORITHM_INDEX = 2;
+
+export const FIRST_ALGORITHM = {
+  id: 'exec-fast-algo',
+  name: 'algorithm 1',
+  description: `description`,
+  filepath: path.join(__dirname, './executionFast_algorithm.py'),
```

Comment:
Maybe let's not duplicate the python algorithms, so comfortably use the file from `fixtures/executions`. Only If you need specific algorithms for your pipelines you can create some.